balena/manager.Dockerfile

```diff
@@ -23,6 +23,7 @@ ENV OR_EMAIL_FROM ${OR_EMAIL_FROM}
 ENV OR_EMAIL_ADMIN ${OR_EMAIL_ADMIN}
 ENV OR_EMAIL_X_HEADERS ${OR_EMAIL_X_HEADERS}
 ENV OR_FIREBASE_CONFIG_FILE ${OR_FIREBASE_CONFIG_FILE:-/deployment/manager/fcm.json}
+ENV OR_GOPACS_PRIVATE_KEY_FILE ${OR_GOPACS_PRIVATE_KEY_FILE:-/deployment/manager/gopacs_private.pem}
```

Comment:
We need to stop setting defaults in the `Dockerfile` as once this is an optional module then we won't be doing this